weather.go

```diff
@@ -1,5 +1,127 @@
 package weather
 
-func Get(location, token string) ([]byte, error) {
-	return nil, nil
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"math"
+	"net/http"
+	"os"
+)
+
+type Weather struct {
```

Comment:
This is fine, but as we're not using most of this struct at the moment, we needn't bother unmarshaling it, need we? We can mention only the fields we actually want, saving a bit of processing time (and a fair bit of code).

Reply:
✅  

go test went from

  > go test .                              
ok      weather 0.478s

to 

> go test .                              
ok      weather 0.231s

Which is quite cool.